test/rendering/cases/webgl-mixed-layers/main.js

```diff
@@ -46,6 +46,7 @@ new Map({
     }),
     new TileLayer({
       source: new DataTile({
+        wrapX: true,
```

Comment:
The WebGL tile layer renderer now will not fetch data outside the tile grid extent if `wrapX` is false.